### DISCUSSION TITLE
2022亏了多少 # 组合季报(2022Q4) · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#132] [Announcements]
# 2022亏了多少 # 组合季报(2022Q4) · 构建我的被动收入

被动收入投资组合2022年Q4季报

https://www.bmpi.dev/money/passive-income-protfolio/2022q4/

**Comment by madawei2699** [comment]
A股下跌的原因主要是上市公司盈利大幅度下降：https://mp.weixin.qq.com/s/DrhIeaM9g8-WQl0erlgC7w

**Reply by madawei2699** [reply]
我们计算的全部A股非金融上市公司盈利增速（滚动四个季度），从2021年二季度高点的67%下降至2022年三季度的-4%，同比增速回落近70个百分点。与之对应的是，宏观经济指标PPI同比增速（PPI是所有宏观指标中与上市公司盈利拟合度最高的指标，可以看做是上市公司盈利增速的代理变量）从2021年10月份的13.5%下降到2022年10月份的-1.3%，增速下降幅度高达14.8%（即1480个bp）。在上市公司盈利快速下行周期，股票市场出现较大幅度的下跌，绝非2022年A股市场的特例，而是具有很强规律性特征的普遍现象。过去几次上市公司盈利增速下行主跌浪中，市场几乎都是熊市，比如2018年、2011年、2008年、2004年。而且这个结论反过来说也基本成立，即A股历史上几次重要的熊市，都是发生在上市公司盈利增速下行的主跌浪中。我们可以这么理解，上市公司短周期盈利增速下行，是对市场最大的利空和伤害，其他的一些利空因素都不足以造成市场如此幅度的下跌，比如2013年当时流动性环境异常收紧，也并没有使得当年股市出现大熊市。

**Reply by madawei2699** [reply]
![IMG_8891](https://user-images.githubusercontent.com/2446612/210483053-404a79d9-b493-41a4-85b1-e72e04c1e4fa.JPG)

**Reply by madawei2699** [reply]
投资大师们的2022年收益👇

![image](https://user-images.githubusercontent.com/2446612/212567299-f0294665-1cda-46e0-a00f-3d818c183f61.jpeg)

**Reply by androo789** [reply]
看来大师们亏的....也很多。。。hhhhhh

**Reply by madawei2699** [reply]
大师们亏的一般是别人的钱😂所以交易自己才是第一负责人。

**Comment by wuwuzhijing** [comment]
请问一下博主，这个博客是使用什么工具搭建的？
